tls plugin isn't loading correctly on 2.8.19

### system info

echo "Haraka | $(haraka -v)"; echo " --- | :--- "; echo "Node | $(node -v)"; echo "OS | $(uname -a)"; 
```
Haraka | Haraka.js — Version: 2.8.19
 --- | :--- 
Node | v6.14.2
OS | Linux mail.example.com 3.10.0-862.3.3.el7.x86_64 #1 SMP Fri Jun 15 04:15:27 UTC 2018 x86_64 
x86_64 x86_64 GNU/Linux
```

echo "openssl | $(openssl version)"
`openssl | OpenSSL 1.0.2k-fips  26 Jan 2017`

### Expected behavior
After updating from 2.8.18, server should continue to work as normal.

### Observed behavior
TLS plugin failes to load correctly with this on the logs:
```
Jun 26 23:34:47 mail.example.com haraka[20396]: [ERROR] [-] [core] tls key /haraka/config/etc/letsencrypt/live/mail.example.com/privkey.pem could not be loaded.
Jun 26 23:34:47 mail.example.com haraka[20396]: [ERROR] [-] [core] tls cert /haraka/config/etc/letsencrypt/live/mail.example.com/fullchain.pem could not be loaded.
[...]
Jun 27 00:10:36 mail.example.com haraka[22101]: [ERROR] [-] [tls] no valid TLS config
```

### Steps to reproduce
Upgrade to 2.8.19 with a valid TLS config.



tls.ini as follows:
# cat tls.ini
```
cert=/etc/letsencrypt/live/mail.example.com/fullchain.pem
key=/etc/letsencrypt/live/mail.example.com/privkey.pem
ciphers=EECDH+AESGCM:EDH+aRSA+AESGCM:EECDH+AES256:EDH+aRSA+AES256:EECDH+AES128:EDH+aRSA+AES128:RSA+AES:RSA+3DES
dhparam=tls/dhparam.pem
```

--- a/plugins/tls.js
+++ b/plugins/tls.js
@@ -23,9 +23,7 @@ exports.shutdown = function () {
 exports.load_tls_ini = function () {
     const plugin = this;
 
-    plugin.cfg = tls_socket.load_tls_ini(function () {
-        plugin.load_tls_ini();
-    });
+    plugin.cfg = tls_socket.load_tls_ini();
 }
 
 exports.advertise_starttls = function (next, connection) {

